doc/releases/migration-guide-3.6.rst

```diff
@@ -279,6 +279,33 @@ Device Drivers and Device Tree
     &fdcan1 {
             clocks = <&rcc STM32_CLOCK_BUS_APB1_2 0x00000100>,
                      <&rcc STM32_SRC_PLL1_Q FDCAN_SEL(1)>;
+            };
+
+* The :dtcompatible:`nxp,kinetis-pit` pit driver has changed it's compatible
+  to :dtcompatible:`nxp,pit` and has been updated to support multiple channels.
+  To configure the individual channels, you must add a child node with the
+  compatible :dtcompatible:`nxp,pit-channel` and configure as below.
+  The :kconfig:option:`CONFIG_COUNTER_MCUX_PIT` has also been to
```

Comment:
Typo here, I assume you meant to say "has also been renamed to"

Reply:
Thank you for noticing, updated!